moto/rekognition/responses.py

```diff
@@ -0,0 +1,63 @@
+"""Handles incoming rekognition requests, invokes methods, returns responses."""
+import json
+
+from moto.core.responses import BaseResponse
+from .models import rekognition_backends
+
+
+class RekognitionResponse(BaseResponse):
+    """Handler for Rekognition requests and responses."""
+
+    @property
+    def rekognition_backend(self):
+        """Return backend instance specific for this region."""
+        return rekognition_backends[self.region]
+
+    def start_text_detection(self):
+        params = self._get_params()
+        video = params.get("Video")
+        client_request_token = params.get("ClientRequestToken")
+        notification_channel = params.get("NotificationChannel")
+        job_tag = params.get("JobTag")
+        filters = params.get("Filters")
+        job_id = self.rekognition_backend.start_text_detection(
+            Video=video,
```

Comment:
These are probably not necessary, considering they are unused in the backend.
If do you want to keep them, they should be in `snake_case`-format

Reply:
Let me give a bit of background in how Moto works:

1. The user calls the boto3 API
2. Boto3 performs some initial validation, such as validating all mandatory parameters are included
3. Boto3 turns the API request into an HTTP request, to send this off to the AWS HTTP API
4. Moto only then steps in, and intercepts this request
5. Moto returns a HTTP response, in the same format as the AWS HTTP API would
6. Boto then transforms the HTTP response into the expected boto3 API response

This means two things:
 - There is no guarantee that the parameters Moto receives, the parameters from the HTTP request, are the same name/format as what was send from the boto3 API.
 - Users will rarely interact with the Moto API,  only with the boto3 API.

In other words, we have free range as to what Moto does internally - as long as we use the correct naming/format for the HTTP request parameters (such as params.get("`JobTag`")) and the response. 

The reason that we happen to use `snake_case` is simply consistency, making it easier for developers and maintainers to read the code.

Reply:
BTW, the `params.get("..."`) is deceivingly simple. As the HTTP request parsing logic is the same across all services, we just expose a few simple functions to retrieve the user-supplied parameters. But there is a lot happening behind the scenes there, to figure out whether that parameter comes from the request body, the querystring, as a path-variable, etc.

Reply:
Thank you so much for your detailed response @bblommers Have amended that now.